ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp

```diff
@@ -35,11 +35,12 @@ void AeroDrone::fetchTelemetryThread()
   // Battery state
   auto battery_state_sub = nh_.subscribe<sensor_msgs::BatteryState>(
       "mavros/battery", 1, boost::bind(&AeroDrone::setBatteryStateCB, this, _1));
+  ros::Rate rate(10);
 
   while (ros::ok())
   {
     ros::spinOnce();
-    rate_.sleep();
+    rate.sleep();
```

Comment:
Are you trying to use rate_

Reply:
There is no such variable `rate_` in telemetry example. What we have used is local variable rate.